### PR TITLE
feat(data): Label assigned to data

### PR DESCRIPTION
**Pull request by mli42**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -169,3 +169,5 @@ outputs
 /data/MNIST
 /data/external
 /data/interim
+/data/H2T/*
+!/data/H2T/*.dvc
```
Comment (ezalos) [inline .gitignore:173]: Nice, I didnt know about the `!xxxx` pattern

File: requirements.txt [hunk 2]
```diff
@@ -8,6 +8,7 @@ awscli>=1.22.24,<1.22.25
 boto3>=1.20.24,<1.20.25
 s3fs>=2021.11.1
 dvc==2.9.5
+opencv-python>=4.5.5.64
```
Comment (ezalos) [inline requirements.txt]: The development depedencies refer to everything you need to contribute and develop the project. 
For example, launching the unit-test, synchronizing the data with our personal bucket.

The project depedencies are the minimum depedencies needed to be able to use the project. Here it's all the librairies imported in our project code.

As such, I would advise to put `opencv-python` in the project depedencies.
I will do a commit to solve it

**Review by ezalos — APPROVED** [review]
`data/video_to_image.py` is really cleanly written, great job!

**Comment by mli42** [comment]
Thank you!

**Pull request merged**